package.json

```diff
@@ -18,7 +18,7 @@
   "main": "app/index.js",
   "repository": "h5bp/generator-server-configs",
   "scripts": {
-    "test": "mocha --timeout 50000"
+    "test": "mocha --timeout 60000"
```

Comment:
cc @sindresorhus for this thoughts. I personally don't think it adds that much value. I do consider tests running for really long a problem though. 10 seconds extra is quite a bunch.